blog/20250218-jupytergis-sync/index.md

```diff
@@ -0,0 +1,63 @@
+---
+title: "JupyterGIS sync meeting 2025-02-18"
+description: |
+  A weekly gathering of JupyterGIS team to discuss our progress and help each other out.
+  Open to all, but this meeting is _short_ and task-focused, so there will not be time
+  for introductions. Please join a core meeting!
+date: "2025-02-18"
+author:
+  - name: "The GeoJupyter community"
+categories:
+  - "JupyterGIS notes"
+tags: [jupytergis-notes]
+---
+
+:::info {.callout-info}
```

Comment:
```suggestion
:::{.callout-info}
```

I wanted this to work with both HackMD and Quarto, but... ugh. I'm sure there's a way, I just need to play more.